### PR TITLE
[CAI-41] ChatInputText atom component

### PR DESCRIPTION
**Pull request by Sebastiano-Bertolin**
#### List of Changes
Add new ChatInputText atom component based on wireframe

#### Motivation and Context
This is an atom component for the chatbot feature

#### How Has This Been Tested?
Tested via Storybook

#### Screenshots (if appropriate):
Screenshot 2024-07-03 at 15 10 50

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      sx={{
+        p: '4px 4px',
+        display: 'flex',
+        alignItems: 'center',
+        width: 'auto',
+      }}
+    >
+      <InputBase
+        fullWidth
+        placeholder='Scrivi un nuovo messaggio'
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx:42]: You should also avoid untranslated strings

**Review by marcobottaro — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx (hunk 1 above)
Comment (Sebastiano-Bertolin) [reply apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx]: Resolved

File: apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    onSubmit(message);
+    setMessage('');
+  };
+
+  const handleSend = () => {
+    onSubmit(message);
+    setMessage('');
+  };
```
[the remark this answers is not in the document]
Comment (Sebastiano-Bertolin) [reply apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx]: Resolved

File: apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+type ChatInputTextProps = {
+  onSubmit: (message: string) => null;
+};
+
+const ChatInputText = ({ onSubmit }: ChatInputTextProps) => {
+  const t = useTranslations();
+  const [message, setMessage] = useState('');
+  const { palette } = useTheme();
+  const iconColor = palette.grey[700];
```
Comment (MarcoPonchia) [inline apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx:14]: This const is used only once i think is better to asing value directly in this case

**Review by MarcoPonchia — CHANGES_REQUESTED** [review]

**Review by MarcoPonchia — APPROVED** [review]

File: .changeset/rotten-pots-whisper.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+"nextjs-website": patch
+"storybook-app": patch
```
Comment (marcobottaro) [inline .changeset/rotten-pots-whisper.md:3]:
```suggestion
"nextjs-website": minor
"storybook-app": minor
```

**Review by marcobottaro — CHANGES_REQUESTED** [review]

**Review by marcobottaro — APPROVED** [review]

**Pull request merged**